setup.py

```diff
@@ -58,10 +58,10 @@ def read(*rnames):
         'Programming Language :: Python :: 2',
         'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.4',
         'Programming Language :: Python :: 3.5',
         'Programming Language :: Python :: 3.6',
         'Programming Language :: Python :: 3.7',
+        'Programming Language :: Python :: 3.8-pre',        
```

Comment:
I do not see 3.8-pre on https://pypi.org/classifiers/

Reply:
Thank you for the link. Just updated the classifier to 3.8 (though it seems a bit odd, as 3.8 final is not yet out).